connectivity/tests/encryption.go

```diff
@@ -23,26 +25,59 @@ const (
 	requestICMPEcho
 )
 
-// getInterNodeIface determines on which netdev iface to capture pkts. In the
-// case of tunneling, we don't expect to see unencrypted pkts on a corresponding
-// tunneling iface, so the choice is obvious. In the native routing mode, we run
-// "ip route get $DST_IP" from the client pod's node.
+// getInterNodeIface determines on which netdev iface to capture pkts.
```

Comment:
Same as with my previous comments - the commit msg needs explanation of the problem.